mivs/site_sections/mivs_applications.py

```diff
@@ -1,12 +1,17 @@
 from mivs import *
 
 
+def allowed_to_submit_round1():
+    return c.BEFORE_ROUND_ONE_DEADLINE or c.HAS_INDIE_ADMIN_ACCESS
```

Comment:
For future reference, this would be a good candidate for a Config mixin, e.g.

```
@Config.mixin
class IndieConfig:
    @property
    def ALLOWED_TO_SUBMIT_ROUND1(self):
        return c.BEFORE_ROUND_ONE_DEADLINE or c.HAS_INDIE_ADMIN_ACCESS
```

This would add ``c.ALLOWED_TO_SUBMIT_ROUND1`` so that you wouldn't need to call this function and pass it to the templates where it's needed.